packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -110,7 +109,6 @@ function convertSummaryIntoContainerSnapshot(createNewSummary: ISummaryTree) {
     const documentAttributes = getDocAttributesFromProtocolSummary(protocolSummary);
     // Currently for the scenarios we have we don't have ops in the detached container. So the
     // sequence number would always be 0 here. However odsp requires to have at least 1 snapshot.
-    assert(documentAttributes.sequenceNumber === 0, "Sequence No for detached container snapshot should be 0");
```

Comment:
Maybe keep it, but assert it's 1 or zero? Or clone it before changing?

Reply:
I think 0 or 1 would be fine.